src/convolution_api.cpp

```diff
@@ -221,20 +221,29 @@ miopenConvolutionCKBackwardWeightsGetWorkSpaceSize(const miopenAlphaBetaCase_t a
 
         assert(in_spatial_dims == miopen::deref(outputTensorDesc).GetNumDims());
 
+        int G    = miopen::deref(convDesc).GetGroupCount();
```

Comment:
General comment to the `miopenConvolutionCKBackwardWeightsGetWorkSpaceSize()` function. As you can see in the other cases, the functions that implement C API contain only thin C/C++ API adaptation layer (and possibly some trivial transformations, like we have for transposed convolutions), and actual job is performed elsewhere. Can you please follow this design principle?

Reply:
[Notice] Resolving a comment at https://github.com/ROCm/MIOpen/pull/2863#discussion_r1640547009 would automatically resolve this one.